ebusd-2.1.x/de/vaillant/15.350.csv

```diff
@@ -5,6 +5,7 @@
 *wi,#install,,,,,"B509","0E",,,,,,
 *ws,#service,,,,,"B509","0E",,,,,,
 *[SW],scan,,,SW,,,,,,,,,
+*[HW],scan,,,HW,,,,,,,,,
```

Comment:
usually the HW version is irrelevant and only the SW decides about features. so please remove all HW parts